Zend/tests/varSyntax/globalNonSimpleVariableError.phpt

```diff
@@ -7,4 +7,4 @@ global $$foo->bar;
 
 ?>
 --EXPECTF--
-Parse error: syntax error, unexpected '->' (T_OBJECT_OPERATOR), expecting ';' or ',' in %s on line %d
+Parse error: syntax error, unexpected '->' (T_OBJECT_OPERATOR), expecting ',' or ';' in %s on line %d
```

Comment:
I'm missing some tests for inheritance behavior. From what I understood (though this is also not specified in the RFC), I'd expect attributes to be preserved if the member is inherited, but not preserved if it is overridden, right?

Reply:
Your assumptions are correct and tests have been added.